SoloProject/server/db.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-console */
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+
+const url = 'mongodb://alandouglas96:Luninance758@ds133328.mlab.com:33328/heroku_w18xw386';
```

Comment:
put this in .env and ignore it this url contains credentials and right now its public !